packages/server/dataloader/atlassianLoaders.ts

```diff
@@ -35,15 +35,10 @@ export const freshAtlassianAuth = (parent: RethinkDataLoader) => {
       const results = await Promise.allSettled(
         keys.map(async ({userId, teamId}) => {
           const atlassianAuth = await getAtlassianAuthByUserIdTeamId(userId, teamId)
-
-          if (!atlassianAuth?.refreshToken) {
-            // Not always an error! For suggested integrations, this won't exist.
-            // sendToSentry(new Error('No atlassian access token exists for team member'), {
-            //   userId,
-            //   tags: {teamId}
-            // })
+          if (!atlassianAuth) {
```

Comment:
The result returned by `getAtlassianAuthByUserIdTeamId` is `Promise<null | AtlassianAuth>` so we definitely need a null check here. On the other hand, refresh token is not nullable anymore so I don't see any reason to keep the refresh token check. I'm happy to improve if I'm missing something though!